### PR TITLE
Added CircleCI deployment to AWS S3

### PR DESCRIPTION
**Pull request by dcgoss**
After @kurtwheeler and I create an S3 bucket for cognoma.org and handle
the IAM permissions this should handle the file sync after a successful
build to the master branch. Based off of:
https://circleci.com/docs/1.0/continuous-deployment-with-amazon-s3/

Also fixed a typo.

### CODE REVIEW

**Comment by dcgoss** [comment]
Kurt and I have everything setup with S3, and cognoma.org is currently being served off S3. This PR would resolve issue #71.

**Comment by dcgoss** [comment]
I currently do not have write access to this repo. Make sure before this gets merged the AWS_BUCKET_PATH environment variable gets set in CircleCI

**Comment by dcgoss** [comment]
Tagging @dhimmel @bdolly for review

File: circle.yml [hunk 1]
```diff
@@ -16,4 +16,11 @@ test:
...
-    - killall --wait sc  # wait for Sauce Connect to close the tunnel
\ No newline at end of file
+    - killall --wait sc  # wait for Sauce Connect to close the tunnel
+
+deployment:
+    production:
+        branch: master
+        commands:
+            - npm run build
+            - aws s3 sync build s3://$AWS_S3_BUCKET_PATH --delete
```
Comment (dhimmel) [inline circle.yml]: Final newline would be nice.

**Pull request merged**